### PR TITLE
note Qt 6.2.4 and use QJsonValue by value instead of reference

### PR DESCRIPTION
**Pull request by groovecoder**
This makes the code build on Mac OS Xcode 12.

## Description

    note Qt 6.2.4 and use QJsonValue by value instead of reference

## Checklist
    
- [ ] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,9 @@ In order to build this application, you need to install a few dependencies.
 
 #### Install Qt6
 
+Note: Currently the build requires 6.2.4. When you run the online installer, select
+6.2.4 version.
```
Comment (brizental) [inline README.md:56]: Is it 6.2.4 or >= 6.2.4? We were just in a meeting where folks said we can build even with 6.4? cc @strseb

Comment (strseb) [reply README.md:56]: Currently only 6.2.4 is considered compatible - I was only starting testing for 6.40 :D

Comment (groovecoder) [reply README.md:56]: I couldn't get it to build on 6.4.0 on Mac.

**Comment by bakulf** [comment]
It seems that this PR is not needed with the latest version of xocde. Closing.

**Pull request closed without merging**